### ISSUE TITLE
Readme is lacking

### ISSUE THREAD

**Issue by GeorgeWL** [dylanpdx/vxtiktok#60]
It's currently an empty file with a header, could we get some documentation there?

**Comment by returnchair** [comment]
yes pls, would be cool. similar to https://github.com/dylanpdx/BetterTwitFix or something like that
